src/js/wrapper.ts

```diff
@@ -25,7 +25,7 @@ export const NATIVE = {
     if (NATIVE.platform === "android") {
       const header = JSON.stringify({
         event_id: event.event_id,
-        sdk_info: event.sdk,
```

Comment:
H: more like a question, what's the data structure of `event.sdk`? were you using the right object but the wrong key only? we attached the `sdk_info` previously. which is a different data structure of `sdk`

Reply:
Yep, I just attached it to the wrong key but used the correct sdk object as `event.sdk` from the JS sdk matches this spec: https://develop.sentry.dev/sdk/event-payloads/sdk/